docs/source/development.rst

```diff
@@ -148,9 +148,9 @@ DXR supports two kinds of integration tests:
    stores the code as a Python string within a subclass of
    ``SingleFileTestCase``. At test time, it instantiates the file on
    disk in a temp folder, builds it, and makes assertions about it. If
-   the ``should_delete_instance`` class variable is truthy (the default), it
+   the ``stop_for_interaction`` class variable is falsy (the default), it
    then deletes the instance. If you want to examine the instance manually for
```

Comment:
For example, we could say "deletes the index" and "If you want to browse the index manually", and it'd be accurate.